README.md

```diff
@@ -3,8 +3,12 @@
     <img src="https://travis-ci.org/precice/dealii-adapter.svg?branch=master" alt="Build status">
 </a>
 
-Coupled structural solvers written with the C++ finite element library deal.II. The coupled codes build on different deal.II tutorials:
-The linear-elastic solver is based on the step-8 tutorial, the nonlinear solver builds on previous work of Jean-Paul Pelteret and Andrew McBride in their deal.II code gallery program 'Quasi-Static Finite-Strain Compressible Elasticity.' Documentation for usage and application can be found in the source code, on the deal.II tutorial documentation and the dealii-adapter wiki pages.
+Coupled structural solvers written with the C++ finite element library deal.II:
+
+- `linear_elasticity` contains a linear-elastic solver based on the step-8 tutorial program of deal.II
+- `nonlinear_elasticity` contains a nonlinear elastic solver, which builds on previous work of Jean-Paul Pelteret and Andrew McBride in their deal.II code gallery program 'Quasi-Static Finite-Strain Compressible Elasticity.' This solver supports shared-memory parallelization.
```

Comment:
Citations here would be useful

Reply:
I added now links to the respective deal.II tutorial program. Is this, what you had in mind?

Reply:
@MakisH Can this be resolved?